prismic-model/js/parts/image.js

```diff
@@ -1,27 +1,38 @@
 // @flow
 
-export default function(label: string) {
+type thumbnailsObject = Array<{
```

Comment:
```suggestion
type Thumbnail = Array<{
```

There's not need to call it an object, we don't really do that elsewhere, and use `UpperCamelCase` for types